Widgets_CDAT/textStyleEditorWidget/TextStyleEditor.py

```diff
@@ -0,0 +1,214 @@
+import vcs
+import text_style_preview
+from PySide import QtCore, QtGui
+
+
+class TextStyleEditor(QtGui.QWidget):
+
+    savePressed = QtCore.Signal(str)
+    
+    def __init__(self):
+        super(TextStyleEditor, self).__init__()
+        self.textObject = None
+
+        wrap = QtGui.QVBoxLayout()
+
+        # Set up vertical align
+        self.va_group = QtGui.QButtonGroup()
+        va_layout = QtGui.QHBoxLayout()
+
+        for alignment in ["Top", "Mid", "Bot"]:
+
+            button = QtGui.QPushButton()
+            button.setText(alignment)
+            button.setCheckable(True)
+            va_layout.addWidget(button)
+            self.va_group.addButton(button)
+
+        self.va_group.buttonClicked.connect(self.updateButton)
+
+        va_box = QtGui.QGroupBox()
+        va_box.setLayout(va_layout)
+        va_box.setTitle("Vertical Align")
+
+        # Set up horizontal group
+        self.ha_group = QtGui.QButtonGroup()
+        ha_layout = QtGui.QHBoxLayout()
+
+        for alignment in ["Left", "Center", "Right"]:
+            button = QtGui.QPushButton()
+            button.setText(alignment)
+            button.setCheckable(True)
+            ha_layout.addWidget(button)
+            self.ha_group.addButton(button)
+
+        self.ha_group.buttonClicked.connect(self.updateButton)
+
+        ha_box = QtGui.QGroupBox()
+        ha_box.setLayout(ha_layout)
+        ha_box.setTitle("Horizontal Align")
+
+        # First row
+        align_angle_row = QtGui.QHBoxLayout()
+        align_angle_row.addWidget(va_box)
+        align_angle_row.addWidget(ha_box)
+        align_angle_row.insertStretch(2, 1)
+
+        # Preview setup
+        self.preview = text_style_preview.TextStylePreviewWidget()
+
+        # Create labels
+        angle_label = QtGui.QLabel()
+        angle_label.setText("Angle:")
+        font_label = QtGui.QLabel()
+        font_label.setText("Font:")
+        size_label = QtGui.QLabel()
+        size_label.setText("Size:")
+
+        # angle dial setup
+        self.angle_slider = QtGui.QDial()
+        self.angle_slider.setRange(90, 450)
```

Comment:
You should stick a comment here explaining why these values, since it looks pretty wonky.